.expeditor/update_hugo_modules.sh

```diff
@@ -0,0 +1,42 @@
+#!/bin/bash
+
+set -evx
+
+# Copy shortcode files from chef product repo
+# to chef/chef-web-docs/layouts/shortcodes
+
+branch="expeditor/${EXPEDITOR_PRODUCT_KEY}_${EXPEDITOR_VERSION}"
+git checkout -b "$branch"
+
+git clone https://x-access-token:${GITHUB_TOKEN}@github.com/chef/${EXPEDITOR_PRODUCT_KEY}/
+
+pushd ${EXPEDITOR_PRODUCT_KEY}
+cp www/layouts/shortcodes/* ../layouts/shortcodes/
+popd
+
+# delete Chef product repo
+
+rm -rf ${EXPEDITOR_PRODUCT_KEY}
+
+# Update the semver version of the chef/chef-workstation module that chef-web-docs will
+# build the workstation docs from.
+# See https://gohugo.io/hugo-modules/use-modules/#get-a-specific-version
+
+hugo mod get github.com/chef/${EXPEDITOR_PRODUCT_KEY}/@${EXPEDITOR_VERSION}
```

Comment:
I think we need to run a `hugo mod tidy` after this command to clean up the old module pinnings. I'll try to illustrate with some command output:
```
➜  hugo mod get github.com/chef/chef-workstation/@0.17.13
go: github.com/chef/chef-workstation 0.17.13 => v0.0.0-20200331232106-41d00ab0699f
2.6.3p62 in ~/d/c/c/chef-web-docs on im/workstation_module ✗

➜  git diff
diff --git a/go.mod b/go.mod
index 07147c705..62cff5633 100644
--- a/go.mod
+++ b/go.mod
@@ -2,4 +2,4 @@ module chef-web-docs

 go 1.14

-require github.com/chef/chef-workstation v0.0.0-20200401145210-19cfbd227952 // indirect
+require github.com/chef/chef-workstation v0.0.0-20200331232106-41d00ab0699f // indirect
diff --git a/go.sum b/go.sum
index f2993aaa7..6e461debe 100644
--- a/go.sum
+++ b/go.sum
@@ -1,2 +1,4 @@
+github.com/chef/chef-workstation v0.0.0-20200331232106-41d00ab0699f h1:LSbDlqh2bEcjEgK4HziGXl2wkAlIfMu4ervqDDxpC3M=
+github.com/chef/chef-workstation v0.0.0-20200331232106-41d00ab0699f/go.mod h1:KZ1lmUGHzTKhRuHdDhB/I2ZEbet++4OTuABWlYb9MlQ=
 github.com/chef/chef-workstation v0.0.0-20200401145210-19cfbd227952 h1:UfAGNN2tPt2G1SZIFeU80JTzYQ3lgVOS4TClAXuVSCE=
 github.com/chef/chef-workstation v0.0.0-20200401145210-19cfbd227952/go.mod h1:KZ1lmUGHzTKhRuHdDhB/I2ZEbet++4OTuABWlYb9MlQ=
```

If I run `hugo mod tidy` you'll notice the frivolous entries in `go.sum` disappear:
```
➜  hugo mod tidy

➜  git diff
diff --git a/go.mod b/go.mod
index 07147c705..62cff5633 100644
--- a/go.mod
+++ b/go.mod
@@ -2,4 +2,4 @@ module chef-web-docs

 go 1.14

-require github.com/chef/chef-workstation v0.0.0-20200401145210-19cfbd227952 // indirect
+require github.com/chef/chef-workstation v0.0.0-20200331232106-41d00ab0699f // indirect
diff --git a/go.sum b/go.sum
index f2993aaa7..0b717e44f 100644
--- a/go.sum
+++ b/go.sum
@@ -1,2 +1,2 @@
-github.com/chef/chef-workstation v0.0.0-20200401145210-19cfbd227952 h1:UfAGNN2tPt2G1SZIFeU80JTzYQ3lgVOS4TClAXuVSCE=
-github.com/chef/chef-workstation v0.0.0-20200401145210-19cfbd227952/go.mod h1:KZ1lmUGHzTKhRuHdDhB/I2ZEbet++4OTuABWlYb9MlQ=
+github.com/chef/chef-workstation v0.0.0-20200331232106-41d00ab0699f h1:LSbDlqh2bEcjEgK4HziGXl2wkAlIfMu4ervqDDxpC3M=
+github.com/chef/chef-workstation v0.0.0-20200331232106-41d00ab0699f/go.mod h1:KZ1lmUGHzTKhRuHdDhB/I2ZEbet++4OTuABWlYb9MlQ=
```